httpserver/src/main/kotlin/tbdex/sdk/httpserver/models/FakeOfferingsApi.kt

```diff
@@ -23,11 +20,7 @@ class FakeOfferingsApi : OfferingsApi {
       payoutUnitsPerPayinUnit = "0.000038",
       payinMethods = listOf(PaymentMethod(kind = "DEBIT_CARD")),
       payoutMethods = listOf(PaymentMethod(kind = "BTC_ADDRESS")),
-      requiredClaims = PresentationDefinitionV2(
```

Comment:
curious why this became null?

Reply:
It was an empty PD with no constraints, so it wasn't actually doing anything